control-plane/grpc/src/operations/volume/traits.rs

```diff
@@ -305,7 +418,12 @@ impl TryFrom<volume::NodeTopology> for NodeTopology {
                     NodeTopology::Explicit(explicit.into())
                 }
             },
-            None => return Err(ReplyError::unwrap_err(ResourceKind::Volume)),
+            None => {
+                return Err(ReplyError::missing_argument(
+                    ResourceKind::Volume,
+                    "node_topology.topology",
```

Comment:
FYI, this `.topology` refers to the grpc's node_topology type, which represents it in this way.